### PR TITLE
Shortform list view

### PR DESCRIPTION
**Pull request by oetherington**
This is a relatively small PR that just adds the new shortform item:
Screenshot 2023-03-14 at 13 38 32
Screenshot 2023-03-14 at 13 38 20

The only place I could find where this exists is the `allPosts` page, but it's possible I missed somewhere else - it should be trivial to drop it in if so.

Currently, clicking takes you to the full shortform on the user's shortform page, but maybe it should just expand in place like the current view does?

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204170919026994) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
The topics feed list view has them

**Comment by jpaddison3** [comment]
I would definitely recommend expanding into place, it's part of the magic of the system currently, according to me.

**Comment by jpaddison3** [comment]
If the lightbulb is the new shortform icon, why not change it next to the shotform section subheader?

**Comment by Will-Howard** [comment]
> If the lightbulb is the new shortform icon, why not change it next to the shotform section subheader?

Controversial but I think I prefer the existing icon. I also think maybe it's not necessary to have the lightbulb icon in the actual list item because there will always be a header saying "shortform" (on both the all posts page and the topic pages) so it should be obvious

File: packages/lesswrong/components/shortform/ShortformListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    maxWidth: SECTION_WIDTH,
+    display: "flex",
+    alignItems: "center",
+    background: theme.palette.grey[0],
+    border: `1px solid ${theme.palette.grey[100]}`,
+    borderRadius: theme.borderRadius.default,
+    padding: 12,
```
Comment (agnesstenlund) [inline packages/lesswrong/components/shortform/ShortformListItem.tsx:17]:
```suggestion
    padding: 8 12,
```

**Comment by agnesstenlund** [comment]
woop this looks great.

agree about the hover previews and padding @Will-Howard 👍  

would also probably use the mini-version of the topic here since this one is _just a bit taller_ than the upvote/agree + is styled similarly (also looks like it has the non-core styling?)
Screenshot 2023-03-16 at 12 37 23

I also realised I forgot to add this design to the shortform, i.e. to move the link icon to the right-handside (only added it to comments)
Screenshot 2023-03-16 at 12 43 09

another thing: wondering if we should either change to lightbulb icon in the title too (where it says Shortform) or just use the old icon on the list items (I know @SharangP was a bit against the bulb icon)

**Comment by oetherington** [comment]
@agnesstenlund Thanks! I've just added the changes:
Screenshot 2023-03-16 at 13 29 27
Screenshot 2023-03-16 at 13 31 04

**Comment by oetherington** [comment]
@agnesstenlund RE the lightbulb icon in the title, I think it doesn't matter because the style will change to this in the all posts page PR (which should be in review at some point today):
Screenshot 2023-03-16 at 13 36 37

One other question that Sarah and I had when we paired on this yesterday: When we expand a shortform it has some margin underneath by default - is this ok or should we remove it?
Screenshot 2023-03-16 at 13 35 07
Screenshot 2023-03-16 at 13 35 14

**Comment by oetherington** [comment]
@agnesstenlund Ok! I've just removed the lightbulb and halved the bottom-margin:
Screenshot 2023-03-16 at 18 09 03
Screenshot 2023-03-16 at 18 14 57

**Comment by agnesstenlund** [comment]
boom perfect! 🚀

**Comment by agnesstenlund** [comment]
looks in the second screenshot like the bulb icon is still showing in the expanded view? or maybe the screenshot was taken before the change to remove it?

**Comment by oetherington** [comment]
Ooh - very well spotted. Fix incoming!

**Comment by oetherington** [comment]
Screenshot 2023-03-16 at 18 39 16
Screenshot 2023-03-16 at 18 39 06

**Pull request merged**